test.hxml

```diff
@@ -11,4 +11,4 @@
 --remap flash:openfl
 
 ## Neko
--cmd neko build/neko_test.n
\ No newline at end of file
+-neko build/neko_test.n
```

Comment:
Done. I just need your feedback on [this comment about pixel diffs](https://github.com/openfl/svg/pull/26#issuecomment-209538570) and we're done here.